ledger/metrics/src/main/scala/com/daml/metrics/api/MetricHandle.scala

```diff
@@ -80,8 +80,12 @@ object MetricHandle {
 
   object Timer {
 
-    trait TimerHandle extends AutoCloseable {
-      def stop(): Unit = close()
+    trait TimerHandle {
+
+      def stop()(implicit
+          context: MetricsContext = MetricsContext.Empty
```

Comment:
Can't the default here silently lead to programming errors?

Reply:
Not for the moment. Currently the main API is without labels, we explicitly use labels only for some specific metrics. 
This will change most likely with the 3.0 release, where labels will be supported everywhere and these defaults will be removed.